packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -78,7 +78,7 @@ export const PostProcessingComponent = defineComponent({
   /** @todo this will be replaced with spatial queries or distance checks */
   reactor: () => {
     const entity = useEntityContext()
-    const rendererEntity = useScene(entity)
+    const rendererEntity = useScene(Engine.instance.viewerEntity)
```

Comment:
why this change? this means any asset panels will not have postprocessing and apply it to the main canvas